lib/appConfig.ts

```diff
@@ -1,32 +1,45 @@
-import { StrategyOptions } from 'passport-github'
+const { URL } = require("url");
```

Comment:
Should we use es6 import statement?
```suggestion
import { URL } from 'url';
```